src/iosource/Manager.h

```diff
@@ -2,30 +2,43 @@
 
 #pragma once
 
+#include "zeek-config.h"
+
 #include <string>
-#include <list>
-#include "iosource/FD_Set.h"
+#include <vector>
+#include <map>
+
+#include "IOSource.h"
+#include "Flare.h"
 
 namespace iosource {
 
-class IOSource;
 class PktSrc;
 class PktDumper;
 
 /**
- * Singleton class managing all IOSources.
+ * Base class for the IOSource Manager. Since the Manager class itself can be
```

Comment:
Are the comments regarding base classes / derived classes still correct here? Looks like we don't actually derive anything from the Manager, and below `Poll` says it must be overridden but it isn't virtual. I suppose this has all changed with the introduction if kqueue?